src/controller/python/OpCredsBinding.cpp

```diff
@@ -468,9 +478,9 @@ PyChipError pychip_OpCreds_AllocateControllerForPythonCommissioningFLow(
 PyChipError pychip_OpCreds_AllocateController(OpCredsContext * context, chip::Controller::DeviceCommissioner ** outDevCtrl,
                                               chip::Controller::ScriptDevicePairingDelegate ** outPairingDelegate,
                                               FabricId fabricId, chip::NodeId nodeId, chip::VendorId adminVendorId,
-                                              const char * paaTrustStorePath, bool useTestCommissioner,
-                                              bool enableServerInteractions, CASEAuthTag * caseAuthTags, uint32_t caseAuthTagLen,
-                                              chip::python::pychip_P256Keypair * operationalKey)
+                                              const char * paaTrustStorePath, const char * dacRevocationSetPath,
```

Comment:
Please do not add this argument and let's configure the commissioner instance separately

Reply:
Makes sense, added another API to configure the revocation set path.